src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -734,5 +768,19 @@ public void SetLabel(string key, long value)
 
 		public void SetLabel(string key, decimal value)
 			=> _context.Value.InternalLabels.Value.InnerDictionary[key] = value;
+
+		private struct DroppedSpanStatsKey
```

Comment:
Implementing `IEquatable<DroppedSpanStatsKey>` would avoid the inherited reflection-based equality approach and could check equality on fields directly.